edit.go

```diff
@@ -53,6 +53,20 @@ func simpleEditor(v *View, key Key, ch rune, mod Modifier) {
 		v.MoveCursor(-1, 0, false)
 	case key == KeyArrowRight:
 		v.MoveCursor(1, 0, false)
+	case key == KeyTab:
+		v.EditNewLine()
+	case key == KeySpace:
+		v.EditWrite(' ')
+	case key == KeyInsert:
+		v.Overwrite = !v.Overwrite
+	case key == KeyCtrlU:
```

Comment:
Fair point! I am not really sure if adding editor specific keybindings are good since that will add a lot of code for little gain.

But I can remove the Ctrl+U and others from the list

Reply:
Agree, that the amount of code for such a small thing is indeed not worth it.  
But we could add this kind of functionality later via a repo or a sub directly with additions.

Reply:
I don't know if it is possible to detect what editor you are in. Perhaps the ENV can tell us. But I'm not really a fan off making that our responsibility. If the program maker wants to make that a feature, he or she can do that.

Does that sound reasonable?

Reply:
Sounds good to me

Reply:
it was my understanding that ctrl + A was a common way of setting the cursor to the beginning of a line, likewise that ctrl + U was a common way of deleting from the cursor to the beginning of the line (and ctrl + E for end of line). I can do it in my terminal, without needing to be in an application. I think it makes sense to include these here. Having said that, these are functions for the simple editor, so I can understand wanting to keep it simple. 

If we remove these, I suppose I would need to re-implement both in lazygit and lazydocker, which is kinda annoying because I'd need to effectively duplicate the code in both repos, fork awesome-gocui (which means the two may diverge again) or use a new repo for gocui-related stuff that's not in awesome-gocui.

Could we perhaps leave the functions there but not reference them in simpleEditor?